### DISCUSSION TITLE
How can I resolve merge conflicts in a GitHub pull request?

### DISCUSSION THREAD

**Discussion by src-1776** [community/community#112137] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm kinda new to this, could someone simply explain how to resolve merge conflicts in a pull request? 

P.S in a hurry for a deadline. ; )

**Comment by supunxiii** [comment, marked as answer]
hello! this can be done easily. so let me break down the steps for you.

1. review the conflict: github will indicate which files have conflicts. click on the file to see the conflicting changes highlighted.

2. locate the conflicts: look for sections of the code marked with "<<<<<<<", "=======" and ">>>>>>>". 

3. choose the correct changes: decide which changes you want to keep. you can edit the code directly in the GitHub interface or choose a text editor locally.

4. commit the changes: once conflicts are resolved commit the changes.

5. complete the merge: if you resolved conflicts locally push the changes to your branch on GitHub. then return to the pull request on GitHub and click the "Resolve Conflicts" button. confirm that the conflicts are resolved and complete the merge.

hope this helped you. if not let me know. good luck!!!
